Source/Core/Core/Slippi/SlippiNetplay.cpp

```diff
@@ -367,7 +419,7 @@ unsigned int SlippiNetplayClient::OnData(sf::Packet &packet, ENetPeer *peer)
 		auto sendTime = ackTimers[pIdx].Front().timeUs;
 		ackTimers[pIdx].Pop();
 
-		pingUs[pIdx] = Common::Timer::GetTimeUs() - sendTime;
+		pingUs[pIdx] = Common::Timer::GetTimeUs() - sendTime - departureDelayUs;
```

Comment:
It doesn't, ping is just calculated as (arrival of ack - departure of pad - departure delay communicated) instead of (arrival of ack - departure of pad). It should be transparent to the rest of the world.
The ping information will be slightly older though ( < 1f ) since the required info is sent with the next pad message instead of being sent instantly.